packages/sdk/viem-account/ledger/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@celo/viem-account-ledger",
+  "version": "0.0.1",
+  "description": "Celo's account helpers",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "author": "Celo",
+  "license": "Apache-2.0",
+  "homepage": "https://docs.celo.org/developer/tools",
+  "repository": "https://github.com/celo-org/developer-tooling/tree/master/packages/sdk/viems-account",
+  "keywords": [
+    "celo",
+    "blockchain",
+    "contractkit"
+  ],
+  "scripts": {
+    "build": "yarn run --top-level tsc -b .",
+    "clean": "yarn run --top-level tsc -b . --clean",
+    "docs": "yarn run --top-level typedoc",
+    "test": "NODE_OPTIONS='--experimental-vm-modules' yarn run --top-level jest --runInBand",
+    "lint": "yarn run --top-level eslint -c .eslintrc.js ",
+    "prepublishOnly": "yarn build"
+  },
+  "dependencies": {
+    "@celo/base": "^6.1.0",
+    "@celo/connect": "^6.0.1",
+    "@celo/ledger-token-signer": "^0.4.0",
+    "@celo/utils": "^7.0.0",
+    "@celo/wallet-base": "^6.0.1",
+    "@celo/wallet-remote": "^6.0.1",
```

Comment:
are these  and the the connect package actually needed? would be best if they were not